tests/safeds_runner/server/test_memoization.py

```diff
@@ -1,11 +1,14 @@
+import sys
 import tempfile
+import time
 import typing
 from datetime import UTC, datetime
 from queue import Queue
 from typing import Any
 
 import pytest
-from safeds_runner.server import pipeline_manager
+from safeds_runner.server import memoization_map, pipeline_manager
```

Comment:
Instead of importing modules with `from..import`, import the declarations that you need inside of them.